Makefile

```diff
@@ -55,3 +61,10 @@ e2e-test-images: manager
 travis-deploy-images: manager
 	docker login -u ${DOCKER_USER} -p ${DOCKER_PASSWORD} ${REGISTRY}
 	./scripts/build-images.sh ${REPOSITORY} true
+
+publish-chart-repo:
+	./scripts/build/checkout-rebase-pages.sh
+	./scripts/build/build-chart-repo.sh
+	./scripts/build/push-to-pages.sh
+
+travis-release-deploy: travis-deploy-images	publish-chart-repo
```

Comment:
`travis-deploy-release`, for consistency with the naming of the other make targets? or maybe just `travis-release`?

Reply:
Updated as you suggested.